src/Main.java

```diff
@@ -2,18 +2,48 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws InterruptedException {
 
         ConsolePrinter.enterDimensionsInstruction();
-         ArrayList<Integer> dimensions = UserInputManager.getGridDimensions();
 
-        CellGrid cellGrid = new CellGrid(dimensions);
 
+        int[] value;
 
-        String[][] newCellGrid = CellGridTranslator.getCellGridAsStringArray(cellGrid);
+        try {
+            value = UserInputManager.turnUserInputIntoDimensions();
+        }
+        catch (IncorrectInputException message){
+            System.err.println(message.getMessage());
+            return;
+        }
 
+        ConsolePrinter.enterLiveCellCoordinates();
+
+        CellGrid cellGrid = new CellGrid(value[0], value[1]);
+
+        try {
+            ArrayList<Coordinates> coords = UserInputManager.turnUserInputIntoCoordinates();
+            cellGrid.setCellToAlive(coords);
+        }
+        catch (IncorrectInputException message){
+            System.err.println(message.getMessage());
+            return;
+        }
+        String[][] newCellGrid;
+
+
+
+        for(int turns = 0; turns < 10; turns++){
+//          print
+            newCellGrid = CellGridTranslator.getCellGridAsStringArray(cellGrid);
```

Comment:
I understand 'CellGridTranslator` is responsible of translating the cell state into `x` or `o`. Do you see `CellGridTranslator` as part of `ConsolePrinter`?

Reply:
there is a bit of an overlap at the moment; probably need to do some refactoring